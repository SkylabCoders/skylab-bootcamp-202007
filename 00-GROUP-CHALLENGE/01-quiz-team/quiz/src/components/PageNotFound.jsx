import React from 'react';
import { Link, Redirect} from 'react-router-dom';
import * as ROUTES from './../config/routes';

function PageNotFound(){
    const condition = false;
    return (
        <section>
            {condition === true && <Redirect to={ROUTES.HOME}/>}
            <h1>Error 404:</h1>
            <h2>Oops!! Page not found!!!!</h2>
            <Link to={ROUTES.HOME}>Navigate to Home</Link>
        </section>
    )
}

export default PageNotFound;