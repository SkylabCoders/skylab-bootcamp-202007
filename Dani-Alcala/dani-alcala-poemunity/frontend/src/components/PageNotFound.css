.page-not-found__container {
  display: -webkit-box;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-orient: vertical;
  -webkit-box-direction: normal;
      -ms-flex-direction: column;
          flex-direction: column;
  -webkit-box-align: center;
      -ms-flex-align: center;
          align-items: center;
}

.page-not-found__message {
  font-size: 30px;
  display: -webkit-box;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-orient: vertical;
  -webkit-box-direction: normal;
      -ms-flex-direction: column;
          flex-direction: column;
  width: 500px;
  height: 500px;
  -webkit-box-pack: center;
      -ms-flex-pack: center;
          justify-content: center;
  -webkit-box-align: center;
      -ms-flex-align: center;
          align-items: center;
}
/*# sourceMappingURL=PageNotFound.css.map */