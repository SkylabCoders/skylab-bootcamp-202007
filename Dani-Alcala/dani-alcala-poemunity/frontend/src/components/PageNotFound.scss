.page-not-found__container {
    display: flex;
    flex-direction: column;
    align-items: center;
}

.page-not-found__message {
    font-size: 30px;
    display: flex;
    flex-direction: column;
    width: 500px;
    height: 500px;
    justify-content: center;
    align-items: center;
}