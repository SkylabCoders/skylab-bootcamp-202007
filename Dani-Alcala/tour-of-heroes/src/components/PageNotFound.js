import React from 'react';
import { Link, Redirect } from 'react-router-dom';


function PageNotFound(props) {
    // const condition = true;
    // props.history.goBack()//hay metodos del BrowserRouter para decir q hacer en ciertos casos
	return (
		<section>
            {/* {condition === true && <Redirect to="/" />} */}
			<h1>Error - 404</h1>
			<p>Nothing to see here</p>
			<Link to="/">Back to Dashboard</Link>
		</section>
	);
}

export default PageNotFound;
