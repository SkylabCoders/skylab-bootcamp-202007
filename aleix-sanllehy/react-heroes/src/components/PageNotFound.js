import React from 'react';
import { Link, Redirect } from 'react-router-dom';

function PageNotFound() {
	const condition = false;
	return (
		<section>
			{condition === true && <Redirect to="/" />}
			<h1>HTTP 404 - Not found</h1>
			<p>Nothing to see here...</p>
			<Link className="button-back hero-element" to="/">
				Back
			</Link>
		</section>
	);
}

export default PageNotFound;
