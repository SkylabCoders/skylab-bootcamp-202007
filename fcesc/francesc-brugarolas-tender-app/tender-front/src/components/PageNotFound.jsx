import React from 'react';
import { Link } from 'react-router-dom';
import ROUTES from '../routes/ROUTES';

function PageNotFound() {
  return (
    <div className="PageNotFound">
      <p>Ooops. The requested page does not exist. Navigate <Link to={ROUTES.home.path}>back</Link> to main page.</p>
    </div>
  );
}

export default PageNotFound;