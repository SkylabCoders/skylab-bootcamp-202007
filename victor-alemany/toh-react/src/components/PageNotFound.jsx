import React from 'react';
import { Link, Redirect, BrowserRouter as Router} from 'react-router-dom';

function PageNotFound(){
    const condition = false;
    return (
        <section>
            <Router>
            {condition === true && <Redirect to="/"/>}
            <h1>Error 404:</h1>
            <h2>Oops!! Page not found!!!!</h2>
            <Link to="/">Navigate to Dashboard</Link>
            </Router>
        </section>
    )
}

export default PageNotFound;