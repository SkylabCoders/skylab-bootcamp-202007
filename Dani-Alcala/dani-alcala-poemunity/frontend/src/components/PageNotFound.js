import React from 'react';
import { Link } from 'react-router-dom';
import "./PageNotFound.scss";


function PageNotFound(props) {
	return (
		<main className="page-not-found__container">
            <section className="page-not-found__message">
			<h1 className="page-not-found__title">Error - 404</h1>
			<p className="page-not-found__text">Nothing to see here</p>
			<Link className="page-not-found__link" to="/">Back to Dashboard</Link>
			</section>
		</main>
	);
}

export default PageNotFound;
