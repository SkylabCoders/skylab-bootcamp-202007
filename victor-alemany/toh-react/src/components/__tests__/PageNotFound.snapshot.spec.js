import renderer from 'react-test-renderer'
import PageNotFound from '../PageNotFound'
import React from 'react'


describe('App snapshot', ()=>{
    const tree = renderer.create(<PageNotFound />);

    it('should match', ()=>{
        expect(tree.toJSON()).toMatchSnapshot();
    })
})
