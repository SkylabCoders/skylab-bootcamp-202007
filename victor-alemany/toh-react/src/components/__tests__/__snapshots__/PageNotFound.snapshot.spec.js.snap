// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot should match 1`] = `
<section>
  <h1>
    Error 404:
  </h1>
  <h2>
    Oops!! Page not found!!!!
  </h2>
  <a
    href="/"
    onClick={[Function]}
  >
    Navigate to Dashboard
  </a>
</section>
`;
